docs/integrations/amazon-aws/amazon-opensearch-service.md

```diff
@@ -8,20 +8,345 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+      "processing": true,
+      "aRN": "arn:aws:es:us-west-1:123456789012:domain/test-domain",
+      "domainId": "123456789012/test-domain",
+      "deleted": false,
+      "domainName": "test-domain",
+      "accessPolicies": {...},
+  "requestID": "12345678-1234-1234-1234-987654321098",
+  "eventID": "87654321-4321-4321-4321-987654321098",
+  "eventType": "AwsApiCall",
+  "recipientAccountId": "123456789012"
+}
+
+```
+
+### **Sample queries**
+
+Successful Events by Event Name
+
+```
+
+account={{account}} region={{region}} namespace=aws/es "\"eventsource\":\"es.amazonaws.com\""
+| json "userIdentity", "eventSource", "eventName", "awsRegion", "sourceIPAddress", "userAgent", "eventType", "recipientAccountId", "requestParameters", "responseElements", "requestID", "errorCode", "errorMessage" as userIdentity, event_source, event_name, region, src_ip, user_agent, event_type, recipient_account_id, requestParameters, responseElements, request_id, error_code, error_message nodrop
+| where event_source = "es.amazonaws.com" 
+| json field=userIdentity "accountId", "type", "arn", "userName"  as accountid, type, arn, username nodrop
+| parse field=arn ":assumed-role/*" as user nodrop 
+| parse field=arn "arn:aws:iam::*:*" as accountid, user nodrop
+| json field=requestParameters "domainName" as domainname nodrop
+| if (isBlank(accountid), recipient_account_id, accountid) as accountid
+| where (tolowercase(domainname) matches tolowercase("{{domainname}}")) or isBlank(domainname)
+| if (isEmpty(error_code), "Success", "Failure") as event_status
+| if (isEmpty(username), user, username) as user
+| count as event_count by event_name
+| sort by event_count, event_name asc
+```
+
+Write Latency by Domain Name (Metrics-based)
+
+```
+
+account={{account}} region={{region}} namespace=aws/es domainname={{domainname}} !nodeid=* metric=WriteLatency statistic = average | avg by domainname 
+```
+
+## **Collecting logs and metrics for the Amazon OpenSearch app**
+
+### **Collecting Metrics for Amazon OpenSearch**
+
+1. Configure a [Hosted Collector](https://help.sumologic.com/docs/send-data/hosted-collectors/configure-hosted-collector/).  
+2. Configure an [Amazon CloudWatch Source for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics/) or [AWS Kinesis Firehose for Metrics Source](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) (Recommended).  
+3. Namespaces. Select aws/es.  
+4. Metadata. Add an account field to the source and assign it a value that is a friendly name/alias to your AWS account from which you are collecting metrics. The account field allows you to query metrics.  
+   ![Metadata][image2]  
+5. Click Save.
+
+### **Collecting Amazon OpenSearch Events using CloudTrail**
+
+1. Add an [AWS CloudTrail Source](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source/) to your Hosted Collector.  
```

Comment:
Can you add the steps similar to the steps in this doc: https://help.sumologic.com/docs/integrations/amazon-aws/dynamodb/#collect-amazon-dynamodb-cloudtrail-logs